Sources/xcodeproj/Utils/BuildSettingsProvider.swift

```diff
@@ -67,10 +69,12 @@ public class BuildSettingsProvider {
         if let variant = variant, let swift = swift, variant == .debug, swift == true {
             buildSettings["SWIFT_OPTIMIZATION_LEVEL"] = "-Onone"
             buildSettings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = "DEBUG"
+            buildSettings["SWIFT_COMPILATION_MODE"] = "singlefile"
         }
         if let variant = variant, let swift = swift, variant == .release, swift == true {
             buildSettings["SWIFT_OPTIMIZATION_LEVEL"] = "-Owholemodule"
-            buildSettings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = "DEBUG"
+            buildSettings["SWIFT_ACTIVE_COMPILATION_CONDITIONS"] = ""
```

Comment:
thanks for the updates! 

we can skip setting this all together as `""` results in an empty string setting as opposed to not being set.
<img width="801" alt="Screen Shot 2019-05-03 at 10 03 27 AM" src="https://user-images.githubusercontent.com/11914919/57128245-b882b900-6d8a-11e9-8eb5-edd1bc56630f.png">

Reply:
Removed as your suggestion @kwridan